addOns/ascanrulesAlpha/src/main/java/org/zaproxy/zap/extension/ascanrulesAlpha/ForbiddenBypassScanRule.java

```diff
@@ -0,0 +1,138 @@
+package org.zaproxy.zap.extension.ascanrulesAlpha;
+
+import org.parosproxy.paros.Constant;
+import org.parosproxy.paros.core.scanner.AbstractAppPlugin;
+import org.parosproxy.paros.core.scanner.Alert;
+import org.parosproxy.paros.core.scanner.Category;
+import org.parosproxy.paros.network.HttpMessage;
+import org.parosproxy.paros.network.HttpStatusCode;
+
+import java.io.IOException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.Arrays;
+import java.util.LinkedList;
+import java.util.List;
+
+/**
+ * Active scan rule for Reverse proxy testing.
+ *
+ * @author Aman Rawat (@theamanrawat)
+ */
+
+public class ForbiddenBypassScanRule extends AbstractAppPlugin {
+
+    private static String HOST = null;
+    private static String PATH = null;
+    private static String SCHEMA = null;
+    private static String FULL_URL = null;
+
+    @Override
+    public int getId() {
+        return 60103;
```

Comment:
Please open a PR against the primary repo: https://github.com/zaproxy/zaproxy/blob/develop/docs/scanners.md to reserve a scan rule ID.